code/game/machinery/camera/camera.dm

```diff
@@ -295,15 +291,10 @@
 			O.reset_perspective(null)
 			to_chat(O, "The screen bursts into static.")
 
-/obj/machinery/camera/proc/triggerCameraAlarm(var/duration = 0)
-	alarm_on = 1
+/obj/machinery/camera/proc/triggerCameraAlarm()
 	motion_alarm.triggerAlarm(loc, src)
 
 /obj/machinery/camera/proc/cancelCameraAlarm()
-	if(wires.IsIndexCut(CAMERA_WIRE_ALARM))
-		return
-
-	alarm_on = 0
 	motion_alarm.clearAlarm(loc, src)

```

Comment:
Are you sure this should be `motion_alarm` and not `camera_alarm`? The latter is unused anywhere else throughout the codebase, besides for borg registration.

Reply:
This was changed 2 years ago by Tigercat's /tg/ cameranet port - and should probably be changed back